controllers/db_utils.go

```diff
@@ -101,6 +251,39 @@ func getDefaultDBName(engine string) *string {
 	}
 }
 
+func getDefaultEngineVersion(engine *string) *string {
+	if engine == nil {
+		return nil
+	}
+
+	switch *engine {
+	case postgres:
+		return pointer.String("13.7")
+	case sqlserverEe, sqlserverSe, sqlserverEx, sqlserverWeb:
+		return pointer.String("15.00.4236.7.v1")
+	case oracleSe2, oracleSe2Cdb:
+		return pointer.String("19.0.0.0.ru-2022-10.rur-2022-10.r1")
+	case mysql:
+		return pointer.String("8.0.28")
+	case mariadb:
+		return pointer.String("10.6.10")
```

Comment:
how often do you feel these are going to be subject to change? Often enough to be worth of pulling out to vars?

Reply:
Moved the strings to constant variables.
AWS started to support these db versions in 2022. We can use them as long as AWS supports them. We don't have to upgrade to the latest versions unless there are specific requirements.